math/mathmore/test/StatFunction.cxx

```diff
@@ -109,6 +109,7 @@ void StatFunction::TestInverse1(RootFinder::EType algoType)
 }
 
 #ifndef _MSC_VER
```

Comment:
Should this not be marked `R__ENABLE_BROKEN_WIN_TESTS` then? We need to fix the code...

Reply:
As I said, the code doesn't even compile...